### ISSUE TITLE
[GLOBAL] Weird default position for Rakata model

### ISSUE THREAD

**Issue by Salk73** [KOTORCommunityPatches/K1_Community_Patch#440]
I noticed that the default standing position for the Rakata model is pretty much off.

It's pretty glaring and I wonder if something can be done about it.

Here it is possible to see the "issue" in action at the end of the conversation with one of the Elders: https://youtu.be/OnJIqHWSquY?t=398

**Comment by DarthParametric** [comment]
No, not without changing the animations and then adjusting the vast majority of conversational Rakata facings. Too much work for too little gain.

**Comment by Salk73** [comment]
Alright.

I don't know whether I expressed myself wrong or if what I had in mind was something out of scope for this patch but I asked Alvar007's opinion and after telling me he understand exactly what I meant with it he took a look and within 24 hours messaged me back with this:

"I've rotated it a little bit and now when transitioning from the talk animation to the standing one and vice versa the model stays almost at the same angle."

I tested it in game and I'm most definitely going to keep it because it makes quite a huge difference.

I'm attaching it here just in case you want to verify yourself and if it's actually K1-CP worthy material, all credits go to Alvar007.

[n_rakata.zip](https://github.com/KOTORCommunityPatches/K1_Community_Patch/files/5579800/n_rakata.zip)
